tedana/tests/test_model_kundu_metrics.py

```diff
@@ -0,0 +1,37 @@
+"""
+Tests for tedana.model.fit
+"""
+
+import pytest
+import numpy as np
+import pandas as pd
+
+from tedana.model import fit
+
+
+def test_smoke_kundu_metrics():
+    """
+    Smoke test for kundu metrics function
```

Comment:
Additionally, could we have a one-sentence description of what the test is doing?